02_bubble/alanhillgemann/main.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	fmt.Fprint(out, bubble([]int{3, 2, 1, 5}))
+}
+
+func bubble(arr []int) []int {
+	for i := 0; i <= len(arr)-1; i++ {
```

Comment:
More idiomatic`for i := 0; i < len(arr); i++ {`

Reply:
Fixed